lib/ood_core/job/adapters/linux_host/launcher.rb

```diff
@@ -80,12 +80,7 @@ def stop_remote_session(session_name, hostname)
 
     call(*cmd, stdin: kill_cmd)
   rescue Error => e
-    raise e unless (
-      # The tmux server not running is not an error
-      e.message.include?('failed to connect to server') ||
-      # The session not being found is not an error
-      e.message.include?("session not found: #{session_name_label}")
```

Comment:
This error message, and the corresponding test case are unused. They come from running `tmux has-session -t the-session-im-looking-for`, but we don't use that API anywhere.